labels/mapping.go

```diff
@@ -1,42 +1,65 @@
 package main
 
-var mapping = map[string][]Label{
-	"mattermost-plugin-giphy": MergeLabels(core, plugin, helpWanted),
+var coreLabels = MergeLabels(pullRequest, issue, helpWanted, docs)
```

Comment:
I would like to keep the core set smaller and usable on every repo. Right now we don't use the issue labels and HW labels on every repo (and I think we won't do this in the future). I would like to only have `pullRequest` here for now.

Reply:
Done, also removed HW from workflow since it's closed-source.